bind/gencffi_func.go

```diff
@@ -109,6 +109,36 @@ func (g *cffiGen) genFuncBody(f Func) {
 
 	g.wrapper.Printf("_cffi_helper.lib.cgo_func_%[1]s(%[2]s)\n", f.id, strings.Join(funcArgs, ", "))
 
+	if f.err {
+		switch nres {
+		case 1:
+			g.wrapper.Printf("if not _cffi_helper.lib._cgopy_ErrorIsNil(cret):\n")
+			g.wrapper.Indent()
+			g.wrapper.Printf("c_err_str = _cffi_helper.lib._cgopy_ErrorString(cret)\n")
```

Comment:
@sbinet 
For about `ffi.string`. I am using this because of Py3 string handling.
It's kind of utility function, which copy the string value from string pointer and join into python string. (Internally, it uses ctypes.)